frontend/src/components/application_spaces/NewApplicationSpace.vue

```diff
@@ -288,18 +288,24 @@
             size="large"
             style="width: 100%"
           >
-            <el-option
-              v-for="item in spaceResources"
-              :key="item.name"
-              :label="item.name"
-              :value="item.id"
-              :disabled="!item.is_available"
-            />
+            <el-option-group
+              v-for="group in spaceResources"
```

Comment:
we don't have space label in open source version

Reply:
done